staging/src/k8s.io/client-go/util/jsonpath/jsonpath_test.go

```diff
@@ -56,12 +56,28 @@ func testJSONPath(tests []jsonpathTest, allowMissingKeys bool, t *testing.T) {
 			t.Errorf("in %s, execute error %v", test.name, err)
 		}
 		out := buf.String()
-		if out != test.expect {
+		if out != test.expect && !elementsExistsInSlice(strings.Split(out, " "), strings.Split(test.expect, " "))  {
```

Comment:
Can you make the output deterministic somehow? Maybe sorting by key? Then you'd be able to remove the method below.

Reply:
Sure. Will implement on sometime and let you know.

Reply:
@apelisse I fear that might introduce problems in runtime due to the reflect.Value. Is it necessary to give a sorted output or can't we go with a function just for a test?

Reply:
I agree with @apelisse, this line should be reverted and the output made deterministic

Reply:
Outputs are now deterministic. Implemented in the recent commits.